Backpack/BarChart/Classes/BarChart.swift

```diff
@@ -22,6 +22,26 @@ import UIKit
 @objc(BPKBarChart)
 public final class BPKBarChart: UIView {
 
+    /// The label to display in the bar chart key for items with a value
+    public var dataKeyLabel: String? {
```

Comment:
Can we enforce specifiying these via an init and hiding the inits inherited from `UIView`?